test/integration/v3000-py2/files/_mapdata/ubuntu-18.yaml

```diff
@@ -48,8 +48,8 @@ values:
       ext_pillar:
       - cmd_yaml: cat /etc/salt/yaml
       - stack:
-          - /path/to/stack1.cfg
-          - /path/to/stack2.cfg
+        - /path/to/stack1.cfg
+        - /path/to/stack2.cfg
```

Comment:
Actually, none of these indent changes should have been done throughout the verification files.  This change actually introduces a `yamllint` violation (even though we're not using it on these files).  Meaning, it would have been better not to make these changes and it would also reduce the changeset of this PR (many files wouldn't have to be touched).